packages/rich-text-editor/src/vaadin-rich-text-editor.js

```diff
@@ -593,6 +593,25 @@ class RichTextEditor extends ElementMixin(ThemableMixin(PolymerElement)) {
       });
     });
 
+    const TAB_KEY = 9;
+
+    editorContent.addEventListener('keydown', (e) => {
+      if (e.key === 'Escape') {
+        this.__tabBindings = this._editor.keyboard.bindings[TAB_KEY];
```

Comment:
Looking at the code, it seems the original binding reference can be lost if the user presses <kbd>Escape</kbd> multiple times.

Reply:
Good catch! I added a check for the `__tabBindings` and also added a test for this case.